iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/VoltagePerReactivePowerControlImpl.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.StaticVarCompensator;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class VoltagePerReactivePowerControlImpl extends AbstractExtension<StaticVarCompensator> implements VoltagePerReactivePowerControl {
+
+    private double slope;
+
+    public VoltagePerReactivePowerControlImpl(StaticVarCompensator svc, double slope) {
+        super(svc);
+        this.slope = slope;
```

Comment:
Is there any restriction for this value? Do we have to implement a `checkSlope` method to ensure this value is in range, and use it in the constructor and the setter?

Reply:
Yes Indeed it should be positive in the load sign convention.